README.md

```diff
@@ -79,6 +79,9 @@ agent = tx.agents.SeqPGAgent(samples=outputs.sample_id,
 Many more examples are available [here](./examples)
   
 ### Installation
+Texar requires tensorflow 1.10.0 or higher (lower than tensorflow 2.0.0a0) and tensorflow_probability 0.3.0 or higher. Please follow the [tensorflow official instructions](https://www.tensorflow.org/install) and [tensorflow_probability official instractions](https://www.tensorflow.org/probability/install) to install the appropriate version.
```

Comment:
Have you tested this? I.e., tested under `TF==1.10` and `tensorflow_probability==0.3.0`

Reply:
Make the text more structured:

Besides the packages in [requirements.txt](./requirements.txt), Texar additionally requires:

  * tensorflow >= 1.10.0 (but <= 2.0). Follow the [tensorflow official instructions](https://www.tensorflow.org/install) to install the appropriate version
  * tensorflow_probability >= 0.3.0. Follow the [tensorflow_probability official instractions](https://www.tensorflow.org/probability/install) to install.

Reply:
Yes, it has been tested. For `TF<1.10` there is some incorrect outputs for some functions.